src/PerfView/PerfViewData.cs

```diff
@@ -493,6 +498,38 @@ public override void Open(Window parentWindow, StatusBar worker, Action doAfter
                     m_singletonStackSource.Viewer.Focus();
             }
         }
+
+        public void OpenOnExistingWorker(Action doAfter = null)
+        {
+            this.OpenOnExistingWorker(GuiApp.MainWindow, GuiApp.MainWindow.StatusBar, doAfter);
+        }
+
+        public void OpenOnExistingWorker(Window parentWindow, StatusBar worker, Action doAfter = null)
```

Comment:
This feels like it should not be public (since it uses StatusBar, which is hard to get).   I would like it to be private if possible.